resources/build_resources.sh

```diff
@@ -13,6 +13,9 @@
 # The path to output all built .py files to: 
 UI_PYTHON_PATH=../python/tk_multi_publish/ui
 
+# The path to where the PySide binaries are installed
+PYTHON_BASE="/Applications/Shotgun.app/Contents/Resources/Python"
```

Comment:
Maybe I am wrong, but didn't we say recently that we should always use the Desktop's Python to build our resources? I'm putting on Hot Topics.